source/_components/stream.markdown

```diff
@@ -26,6 +26,47 @@ To enable this component, add the following lines to your `configuration.yaml` f
 stream:
 ```
 
+<p class='note'>
+Depending on your setup, you might need to set a base URL (`base_url`) inside the [http component](/components/http/) or in the parameters of this component.
```

Comment:
Should this be here?

Reply:
I rather think that all these changes in the stream markdown file do not even belong to this PR 🤔